app/Controllers/userController.php

```diff
@@ -516,6 +526,30 @@ public function deleteAction() {
 		Minz_Request::forward($redirect_url, true);
 	}
 
+	public function switchAdminAction() {
+		if (!FreshRSS_Auth::hasAccess('admin')) {
```

Comment:
Only POST should be allowed there, to avoid CSRF attacks

Reply:
Noted. I'll add that.